partcad-ide-vscode/src/PartcadExplorer.ts

```
@@ -0,0 +1,316 @@
+        if (element) {
+            if (element.name in this.packages) {
+                return Promise.resolve(this.expandItems(this.packages[element.name]));
+            } else {
+                return this.getPackageContents(element.name);
+            }
+        }
+
+        return Promise.resolve(this.expandItems(this.packages['/']));
+    }
+
+    private _onDidChangeTreeData: vscode.EventEmitter<PartcadItem | undefined | null | void> = new vscode.EventEmitter<
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)